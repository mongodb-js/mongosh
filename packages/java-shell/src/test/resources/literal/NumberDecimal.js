// command checkResultClass
new NumberDecimal("24")