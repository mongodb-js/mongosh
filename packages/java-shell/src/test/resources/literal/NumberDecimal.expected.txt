Decimal128Result: { "$numberDecimal" : "24"}